tests/integration/api/xai/test_api_xai_sanity.py

```diff
@@ -4,13 +4,13 @@
 
 import os
 import os.path as osp
+import tempfile
 
 import pytest
 import torch
 
-from otx.algorithms.classification.tasks import (
+from otx.algorithms.classification.tasks import (  # ClassificationOpenVINOTask,
```

Comment:
I believe, this comment was left by accident

Reply:
Yeah, it is possible to remove it, but if you check below there is commented OV task. I will fix it later. So `# ClassificationOpenVINOTask` is kept for it.